examples/run.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # fetch some morphologies from neuromorpho.org
-python get_morphologies.py
+#python get_morphologies.py
```

Comment:
Maybe you can run it only id the `morphology` folder does not exist?

Reply:
not sure why I commented this lol

Reply:
:smile: